packages/cli/src/commands/domain.ts

```diff
@@ -0,0 +1,328 @@
+import {
+  createDomain,
+  deleteDomain,
+  getDomains,
+  updateDomain,
+} from "@/utils/get-domains";
+import { handleError } from "@/utils/handle-error";
+import { logger } from "@/utils/logger";
+import chalk from "chalk";
+import { Command } from "commander";
+import Configstore from "configstore";
+import ora from "ora";
+import prompts from "prompts";
+import { z } from "zod";
+
+const domainOptionsSchema = z.object({
+  slug: z.string().min(3, {
+    message: "Please provide a valid slug",
+  }),
+});
+
+export const domain = new Command()
+  .name("domain")
+  .description("Configure your workspace domain")
+  .action(async () => {
+    const spinner = ora("Fetching domains").start();
+
+    try {
+      const showDomains = async () => {
+        const domains = await getDomains();
+        if (domains) spinner.stop();
+
+        return [
+          ...domains.map((domain) => ({
```

Comment:
@sujjeee You can use `/api/domains/default` to fetch the default domains enabled for a workspace.

Reply:
should i remove domain related command?

Reply:
Let's remove it and keep the first version as simple as possible.